src/editing_tools/AdjustTool.vala

```diff
@@ -419,28 +419,24 @@ public class EditingTools.AdjustTool : EditingTool {
 
         /* set up temperature & tint */
         var temp_trans = (TemperatureTransformation) transformations.get_transformation (
-            PixelTransformationType.TEMPERATURE
-        );
+            PixelTransformationType.TEMPERATURE);
```

Comment:
Hmm, I don't think we need to change this :thinking:

Reply:
Oh okay. Thanks